mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule ConfigCat.MixProject do
   def project do
     [
       app: :config_cat,
-      version: "0.1.0",
+      version: "1.0.0",
```

Comment:
Can we hold off on bumping the version number for now? We might want to release a pre-1.0 test version before we officially go 1.0.

Reply:
I can revert it but I thought it would be weird to have a branch called v1.0.0 and the package itself with v0.1.0. We can also review this at any time before emerging it to `main` branch, right?

Reply:
Yeah, we can review at any time.